internal/runners/clean/stop.go

```diff
@@ -0,0 +1,47 @@
+package clean
+
+import (
+	"github.com/ActiveState/cli/internal/appinfo"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/exeutils"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/installation"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+)
+
+func stopServices(cfg configurable, out output.Outputer, ignoreErrors bool) error {
+	cleanForceTip := locale.Tl("clean_force_tip", "You can re-run the command with the [ACTIONABLE]--ignore-errors[/RESET] flag.")
+
+	// On Windows we need to halt the state tray and the state service before we can remove them
+	svcInfo := appinfo.SvcApp()
+	trayInfo := appinfo.TrayApp()
+
+	// Todo: https://www.pivotaltracker.com/story/show/177585085
+	// Yes this is awkward right now
+	if err := installation.StopTrayApp(cfg); err != nil {
+		if ignoreErrors {
+			return errs.AddTips(
+				locale.WrapError(err, "clean_stop_tray_failure", "Cleanup interrupted, because a running {{.V0}} process could not be stopped.", trayInfo.Name()),
+				cleanForceTip)
+		} else {
+			out.Print(locale.Tl("clean_stop_tray_warning", "[ERROR]Failed to stop running {{.V0}} process.[/RESET] Continuing anyways, because --force flag was provided.", trayInfo.Name()))
+		}
```

Comment:
I suggest making this !ignoreErrors and dropping the else. And the same below.

Reply:
:facepalm: